system/lib/html5/emscripten_wget.c

```diff
@@ -1,10 +1,13 @@
+#if ASYNCIFY // emscripten_wget requires asyncify
+
```

Comment:
I don't think this macro is ever defined when building this library so I don't think this can work.